core/src/ops/math/mod.rs

```diff
@@ -52,6 +54,125 @@ where
 bin_to_super_type!(mul, Mul,
                    cost: |dt| tvec!((Cost::FMA(dt), 1)),
                    declutter: declutter_mul,
+                   eval_in_a: |a: &mut Tensor, b: &Tensor| -> TractResult<bool>{
```

Comment:
any reason not no make this a function ? if only for the stake of having rustfmt processing it :)